db/re/item_db.conf

```diff
@@ -84091,7 +84091,6 @@ item_db: (
 	Buy: 2
 	Weight: 100
 	Upper: 63
-	Delay: 1000
```

Comment:
why remove this? this delay still applies in aegis
12732 EFST_REUSE_STORMBLAST 1			// ÀÌ´Ù ·é ½ºÅæ (ÀÌ´Ù -> Æä¸£¾²·Î ÀÌ¸§ÀÌ ¹Ù²î¾ú½À´Ï´Ù)